README.md

```diff
@@ -2,6 +2,25 @@
 
 Cleisthenes is PBFT consensus library adaptable to any application.
 
+## Architecture
+
+### Logical View
+
+<p align="center">
+	<img src="./img/cleisthenes-architecture-logical-view.png" ></img>
+</p>
+
+**PBFT** basically communicate with **Application**. Whenever Application needs consensus with its network members, **Application** sends request to **PBFT**.
+
+If one of members in the network need to gossiping message to the network such as notifing changed leader or broadcasting byzantine node, we can gossiping those messages through **Messaging**. 
```

Comment:
According to my suggestion, `one of members` should be change to `one of nodes`.